src/readinglistbar.cpp

```diff
@@ -122,6 +124,20 @@ void ReadingListBar::onExport()
         app->showMessage(gt("export-reading-list-error"), gt("error-title"), QMessageBox::Information);
 }
 
+void ReadingListBar::onImport()
+{
+    auto app = KiwixApp::instance();
+    auto library = app->getLibrary();
+    QString fileName = QFileDialog::getOpenFileName(app->getMainWindow(),
+                                                    gt("open-file"),
+                                                    QString(), "(*.xml)");
```

Comment:
Please make the filter string more user-friendly, change it to `XML files (*.xml)`

Reply:
@veloman-yunkan do we need to translate "XML files" or does Qt does it automatically?

Reply:
That's a good question. But I guess that the answer to it is negative. So maybe it is better to keep the filter language neutral.